packages/ui/src/coreModules/form/components/fields/Date/RangeDate/index.js

```diff
@@ -260,7 +260,11 @@ class DateRange extends Component {
                 setYearInputRef={this.setStartYearInputRef}
               />
             </div>
-            {dateType !== SINGLE && <div style={dateTypeStyle}>{'–'}</div>}
+            {dateType !== SINGLE && (
+              /* eslint-disable react/jsx-curly-brace-presence */
```

Comment:
is this really needed? can we replace `{'-'}` with just `-`? 

this commit should be fixuped with the https://github.com/DINA-Web/dina-collections/pull/283/commits/54322b2a4713a09eb4a8103609921c12773fb2b0